assets/scss/6-components/button/_button.scss

```diff
@@ -24,6 +24,12 @@
   text-transform: uppercase;
   transition: opacity 0.3s;
 
+  &:disabled,
+  &[disabled="true"],
+  &[disabled="disabled"] {
+    opacity: .4;
+  }
+
```

Comment:
good catch alls!

Reply:
I realized I didn't account for the edge case of `disabled="false"`. So updated to this:
![Screen Shot 2019-08-08 at 3 43 26 PM](https://user-images.githubusercontent.com/2243202/62736618-612c5e80-b9f3-11e9-8969-a2c3b565c5df.png)

Reply:
💅